docs/self-hosted-appcircle/install-server/linux-package/configure-server/container-engine-network-subnet.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Configure Container Engine Network Subnet
+sidebar_label: Configure Container Engine Network Subnet
+description: Configure the Appcircle server to use a specific network subnets for the containers.
+tags: [docker, self-hosted, network]
+sidebar_position: 50
+---
+
+import SpacetechExampleInfo from '@site/docs/self-hosted-appcircle/install-server/linux-package/configure-server/_spacetech-example-info.mdx';  
+import RestartAppcircleServer from '@site/docs/self-hosted-appcircle/install-server/linux-package/configure-server/_restart-appcircle-server.mdx';  
+import DowntimeCaution from '@site/docs/self-hosted-appcircle/install-server/linux-package/configure-server/_appcircle-server-downtime-caution.mdx';  
+
+
+## Overview
+
```

Comment:
_**Additional Section**_:
- Default network settings `enabled: false`, which means default subnet for Podman/Docker
- In order to revert custom subnet configuration: Delete the network settings section in yaml or update `enabled: false` to keep the subnet info passive in yaml